spotbugs/src/main/java/edu/umd/cs/findbugs/util/MutableClasses.java

```diff
@@ -137,6 +138,14 @@ private boolean computeMutable() {
 
         private boolean looksLikeASetter(Method method) {
             final String methodName = method.getName();
+            // If the method throws an UnsupportedOperationException then we ignore it.
+            ExceptionTable exceptions = method.getExceptionTable();
+            if (exceptions != null) {
```

Comment:
These two `if` conditions could be combined.

Reply:
This conversation can be resolved, since the if conditions got combined in https://github.com/spotbugs/spotbugs/pull/1672/commits/d30be689a17cced410cdc366aec91d5f706c943b.